planemo/commands/cmd_shed_diff.py

```diff
@@ -53,7 +59,7 @@ def cli(ctx, path, **kwds):
     def diff(realized_repository):
         working = tempfile.mkdtemp(prefix="tool_shed_diff_")
         try:
-            shed.diff_in(ctx, working, realized_repository, **kwds)
+            return shed.diff_in(ctx, working, realized_repository, **kwds)
```

Comment:
It is the last statement in the diff - it is wrapped in a finally not exception handling so this ``return`` is the only place to return from.